lib/blacklight/component.rb

```diff
@@ -3,6 +3,12 @@
 module Blacklight
   class Component < ViewComponent::Base
     class << self
+      def inherited(subclass)
+        # Workaround for https://github.com/ViewComponent/view_component/issues/1565
+        subclass.config = ViewComponent::Base.config
+        super
+      end
```

Comment:
```suggestion
      # Workaround for https://github.com/ViewComponent/view_component/issues/1565
      def config
        @config ||= ViewComponent::Config.defaults.merge(ViewComponent::Base.config)
      end
```

Reply:
I don't love this, either - but it's a safe copy that would capture the base config. It doesn't address the question for me of whether this config's entry point should be in `config.blacklight.view_component` or what.

Reply:
I think this also means we have to pin ViewComponent to `~> 2.65`